src/third_party/dartino/object_memory_mark_sweep.cc

```diff
@@ -409,18 +416,16 @@ void FixPointersVisitor::do_roots(Object** start, int length) {
 }
 
 // This is faster than the builtin memmove because we know the source and
```

Comment:
Are you sure?

Reply:
It was true 5 years ago, when the function did something slightly different :-)

Added TODO to check if it's still true.